build/helper/documentation_helper.py

```diff
@@ -469,6 +469,22 @@ def as_rest_table(data, header=True):
 
 
 # Unit Tests
+
+
+def _remove_trailing_whitespace(s):
+    '''Removes trailing whitespace and empty lines in multi-line strings. https://stackoverflow.com/a/17350806/316875'''
+    return re.sub(r'\s+$', '', s, flags=re.M)
```

Comment:
Does this remove all empty lines? If it does then that means there could be breaking changes that these tests wouldn't catch. 0 empty lines does not mean the same as 1 empty line, but 1 or more empty lines do mean the same thing.

I think this is still better than just asserting on the two multiline strings being equal, so I would be okay with accepting this and making it better later, if needed.

Reply:
Pain in the neck...
I will merge as is because it's useful and opened https://github.com/ni/nimi-python/issues/451 to follow up.